crates/chia-protocol/src/pot_iterations.rs

```diff
@@ -23,15 +23,13 @@ fn div_catch_error(a: u64, b: u64) -> Result<u64> {
 pub fn is_overflow_block(
     num_sps_sub_slot: u32,
     num_sp_intervals_extra: u8,
-    signage_point_index: u32,
+    signage_point_index: u8,
```

Comment:
why is num_sps_sub_slot u32 ?

Reply:
I believe this is mis-typed in `ConsensusConstants`.
https://github.com/Chia-Network/chia-blockchain/blob/main/chia/consensus/default_constants.py#L17